airflow/providers/google/cloud/operators/datafusion.py

```diff
@@ -826,10 +826,8 @@ def __init__(
 
         if success_states:
             self.success_states = success_states
-            self.pipeline_timeout = pipeline_timeout
```

Comment:
Instead of removing it, we should have the hook use the pipeline_timeout when waiting for the states instead of hard-coded timeout

Reply:
sorry my bad, forgot to add that back in, should be there now, this was exactly the change I was trying to resolve

Reply:
I think I fixed this with my change on line 826 :)